scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -123,6 +123,26 @@ trait ClassLikeSupport:
           .map { _ =>
             parseMethod(dd.symbol, kind = Kind.Given(getGivenInstance(dd).map(_.asSignature), None))
           }
+          
+      case dd: DefDef if !dd.symbol.isHiddenByVisibility && dd.symbol.isExported =>
+        val exportedTarget = dd.rhs.flatMap { 
+          case a: Apply => Some(a.fun)
+          case s: Select => Some(s)
+          case _ => None
+        }.map { 
+          case s: Select => s
+        } 
+        val functionName = exportedTarget.fold("instance")(_.name)
+        val instanceName = exportedTarget.fold("function")(_.qualifier.asInstanceOf[Select].name)
```

Comment:
Does it need to a `Select`? What if we have 
```
object A:
  def a = 1  

object B extends App:                                                                                                                                                                      
     export A._
     
```

Reply:
There was a wrong assumption that led to exception breaking the scala3doc generation while having object. Fixed it though